sagemaker-debugger/tensorflow_action_on_rule/detect_stalled_training_job_and_stop.ipynb

```diff
@@ -4,30 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p>"The unique prefix" --&gt; "A unique prefix".</p><p> </p><p>" it will fallback to safe mode and not take action .. " --&gt; " it falls back to safe mode and does not stop the training job."</p><p> </p><p>"Such rule evaluation process goes on in parallel while the training jobs are running" --&gt; can you replace "Such" with something else, maybe "This"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Feedback incorporated